src/navigation/services/buy-crypto/screens/BuyCryptoOffers.tsx

```diff
@@ -438,6 +440,51 @@ const BuyCryptoOffers: React.FC = () => {
         ? amount
         : dispatch(calculateAltFiatToUsd(amount, fiatCurrency)) || amount;
 
+    const currencyLimitsrequestData: MoonpayGetCurrencyLimitsRequestData = {
+      currencyAbbreviation: getMoonpayFixedCurrencyAbbreviation(
+        coin.toLowerCase(),
+        selectedWallet.chain,
+      ),
+      baseCurrencyCode: offers.moonpay.fiatCurrency.toLowerCase(),
+      paymentMethod: _paymentMethod,
+      areFeesIncluded: true,
+      env: moonpayEnv,
+    };
+
+    try {
+      const moonpayCurrencyLimitsData = await moonpayGetCurrencyLimits(
+        currencyLimitsrequestData,
+      );
+      offers.moonpay.amountLimits = {
+        min: moonpayCurrencyLimitsData.baseCurrency.minBuyAmount,
+        max: moonpayCurrencyLimitsData.baseCurrency.maxBuyAmount,
+      };
+
+      if (
+        (offers.moonpay.amountLimits.min &&
+          offers.moonpay.fiatAmount < offers.moonpay.amountLimits.min) ||
+        (offers.moonpay.amountLimits.max &&
+          offers.moonpay.fiatAmount > offers.moonpay.amountLimits.max)
+      ) {
+        offers.moonpay.outOfLimitMsg = t(
+          'There are no Moonpay offers available, as the current purchase limits for this exchange must be between and',
```

Comment:
this looks like a typo? should it be interpolated 

example -> between ${min} and ${max}